cmd/loraserver/main.go

```diff
@@ -420,6 +420,11 @@ func main() {
 			EnvVar: "GW_MQTT_PASSWORD",
 		},
 		cli.StringFlag{
+			Name:   "gw-mqtt-ca-file",
```

Comment:
without the `gw` at the beginning? 

Are you sure? Because all the other flags include the `gw` so I kept consistency with your convetiions...